airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -227,45 +236,43 @@ def create_empty_table(self,  # pylint: disable=too-many-arguments
         :type num_retries: int
         :return: None
         """
-        service = self.get_service()
-
-        project_id = project_id if project_id is not None else self.project_id
+        if num_retries:
+            warnings.warn("Parameter `num_retries` is deprecated", DeprecationWarning)
```

Comment:
Is it now just ignored? (Because it can never fail?)

Reply:
This parameter is supported by the old library. The new library uses a more complex configuration. https://github.com/googleapis/python-api-core/blob/master/google/api_core/retry.py#L242-L250

Reply:
I've added the retry object.